setup.py

```diff
@@ -9,6 +9,13 @@
 import chainerx_build_helper
 
 
+if sys.version_info[0] == 2:
```

Comment:
Could you remove this block? I think `python_requires` is sufficient and it is also consistent with https://github.com/cupy/cupy/pull/2766.

Reply:
I removed the block